admin/src/templates/export.html

```diff
@@ -14,5 +14,8 @@
   <li role="presentation" ui-sref-active="active" mm-auth="can_export_dhis">
     <a ui-sref="export.dhis" translate>DHIS2</a>
   </li>
+  <li role="presentation" ui-sref-active="active">
+    <a ui-sref="export.user-devices" translate>User devices</a>
```

Comment:
I'd suggest using translation keys. For example `export.tabs.user_devices`
That way we don't need to escape the space, instead of `User\ devices =` we can have `export.tabs.user_devices = User devices`